easybuild/easyconfigs/s/SEED/SEED-2015.eb

```diff
@@ -0,0 +1,36 @@
+##
+# This is a contribution from Phoenix HPC Service, The University of Adelaide, Australia
+# Homepage: https://www.adelaide.edu.au/phoenix/
+#
+# Copyright::   Copyright 2014-2017 adelaide.edu.au/phoenix
+# Authors::	Robert Qiao <robert.qiao@adelaide.edu.au>, Exequiel Sepulveda <exequiel.sepulveda@gmail.com>
+# License::     Artistic-v2.0
+#
+# Notes::
+##
+
+easyblock = 'CmdCp'
+
+name = 'SEED'
+version = '2015'
```

Comment:
this is elegant in style, yet I'm afraid there is no `--try-*` escape path in case someone wishes to advance a version,
since `--try-amend=commit=bla` goes at the end of a modified easyconfig and it won't be picked up correctly.

any feedback on that?